test/test-windows-make.js

```diff
@@ -72,14 +74,15 @@ function quote (path) {
   if (path.includes(' ')) {
     return `"${path}"`
   }
+  return path
 }
 
 describe('windows-cross-compile', function () {
   it('build simple node-api addon', async function () {
     if (process.platform !== 'win32') {
       return this.skip('This test is only for windows')
     }
-    const env = getEnv('win-clang')
```

Comment:
Can we skip the test if wasi sdk is not installed? Otherwise we will need to install wasi-sdk for gyp-next repo as well.

Reply:
It will skip if `WASI_SDK_PATH` not exist, so gyp-next no need to change.

https://github.com/nodejs/node-gyp/blob/c0b57d2883dd3eabc0886b1fa0597e9b722a81bb/test/test-windows-make.js#L54
https://github.com/nodejs/node-gyp/blob/c0b57d2883dd3eabc0886b1fa0597e9b722a81bb/test/test-windows-make.js#L85-L88